### PR TITLE
Bug #107 G$ Rewards - Value in USD does not correspond to G$ price.

### LINKED ISSUE #107: G$ Rewards - Value in USD does not correspond to G$ price.
**Bug Description**
Rewards value doesn't correspond to real price!

Captura de pantalla 2021-12-28 a las 13 31 16

**Steps to Reproduce**
1. Stake into the GoodStake positions
2. Go to portfolio, after a certain time earning G$rewards, check that value in USD does not correspond to proper price
...

**Expected Behavior**
680132 G$s *0,00016 USD/G$ = 111.54 G$

**Additional Context**
Add any other context about the problem here (screenshots, whether the bug only occurs only in certain mobile/desktop/browser environments, etc.)

### PR DESCRIPTION
**Pull request by Devartjoker**
# Description

- Add missing G$ logo on Stake page for Kovan network
- Fix floating point misplacement on Portfolio page by multiplying estimate $ values by 100
About # (link your issue here)
Resolves #107

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

# Checklist:
- [x] PR title matches follow: (Feature|Bug|Chore) Task Name
- [x] My code follows the style guidelines of this project
- [x] I have followed all the instructions described in the initial task (check Definitions of Done)
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I have added reference to a related issue in the repository
- [x] I have added a detailed description of the changes proposed in the pull request. I am as descriptive as possible, assisting reviewers as much as possible.
- [ ] I have added screenshots related to my pull request (for frontend tasks)
- [ ] I have pasted a gif showing the feature.
- [ ] @mentions of the person or team responsible for reviewing proposed changes

### CODE REVIEW

File: src/components/PortfolioTableRow/index.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ function PortfolioTableRow({ stake, onWithdraw }: PortfolioTableRowProps) {
                     {stake.rewards.reward && stake.rewards.reward.claimed.currency.symbol} <br />~
                     {stake.rewards.reward$ && stake.rewards.reward$.claimed
                         .add(stake.rewards.reward$.unclaimed)
+                        .multiply(100)
```
Comment (sirpy) [inline src/components/PortfolioTableRow/index.tsx:52]: this is a hack.
the fix should be fixing reward$.claimed and reward$.unclaimed, why are they not in the right precision?
check staking.ts

File: src/pages/gd/Portfolio/index.tsx [hunk 2]
```diff
@@ -94,7 +94,7 @@ const Portfolio = () => {
                             {data?.aggregated?.rewardsG$.currency.symbol}
                             <br />
                             <small>
-                                ~{data?.aggregated?.rewardsG$$.toFixed(2, { groupSeparator: ',' }) ?? '0.00'}$
+                                ~{data?.aggregated?.rewardsG$$.multiply(100).toFixed(2, { groupSeparator: ',' }) ?? '0.00'}$
```
Comment (sirpy) [inline src/pages/gd/Portfolio/index.tsx:97]: fix in the source

File: src/pages/gd/Portfolio/index.tsx [hunk 3]
```diff
@@ -134,7 +134,7 @@ const Portfolio = () => {
                             {data?.aggregated?.rewardsG$Unclaimed.currency.symbol}
                             <br />
                             <small>
-                                ~{data?.aggregated?.rewardsG$Unclaimed$.toFixed(2, { groupSeparator: ',' }) ?? '0.00'}$
+                                ~{data?.aggregated?.rewardsG$Unclaimed$.multiply(100).toFixed(2, { groupSeparator: ',' }) ?? '0.00'}$
```
Comment (sirpy) [inline src/pages/gd/Portfolio/index.tsx:137]: fix the source of the issue

**Review by sirpy — CHANGES_REQUESTED** [review]

**Comment by YpetrovAJ** [comment]
@sirpy check the last commit here

**Pull request merged**